parts/k8s/kubernetesconfigs.sh

```diff
@@ -183,6 +183,42 @@ configureK8s() {
     "providerKeyVersion": ""
 }
 EOF
+    if [[ "${TARGET_ENVIRONMENT,,}" == "azurestackcloud"  ]]; then 
+        echo "AUTHENTICATION_METHOD: ${AUTHENTICATION_METHOD}"
```

Comment:
Do we need these echo statements? There is an upper bound on the CSE payloads in an ARM template, so let's omit them if we can. If we want to display this in the provision log, let's enable `set -x` for any commands that are not private. Make sure we put `set +x` before any output that should not be printed to stdout and captured in logs.

Reply:
@jackfrancis  Thanks for the instructions about set -x and set +x
I removed the echo command here since I could find the value in the ARM template

and set +x already is present at line 144 to protect the secrets. set -x is executed at line 220 to enable the logs.